torchx/schedulers/utils/ray_driver.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+import dataclasses
+import ray
+from typing import Any, Dict, Iterable, List, Optional, Set, Type
+import logging
+import subprocess
+from ray_common import RayActor
+
+@ray.remote
+class CommandActor:
+    def __init__(self, command, env):
+        self.command = command
+        self.env = env
+
+    def run_command(self):
+        return subprocess.run(self.command, env = self.env)
+
+def load_actor_json(filename : str) -> List[Dict]:
+    with open(filename) as f:
+        actor = json.load(f)
+        actor = json.loads(actor)
+        return actor
+
+if __name__ == "__main__":
+
+    # Move to scheduler
+    actor1 = RayActor("resnet", ["echo", "hello resnet"])
+    actor2 = RayActor("bert", ["echo", "hello bert"])
+    serialize([actor1, actor2])
+
+    logging.basicConfig(filename='driver.log', encoding='utf-8', level=logging.DEBUG)
+
+
+    # On driver.py
+    logging.debug("Reading actor.json")
+    actors_dict = load_actor_json('actor.json')
+    pgs = []
+    ray.init(address="auto", namespace="torchx-ray")
+
+    for actor_dict in actors_dict:
+
+        bundle = {"CPU": actor_dict["num_cpus"], "GPU": actor_dict["num_gpus"]}
```

Comment:
I would advise getting @amogkam's feedback on the rest of `ray_driver.py`. We should pay particular attention to the edge cases. How do we handle failures? For instance what happens if an actor or a process group initialization raises an error? How do we communicate the root cause back to the user?